src/app/cluster/details/cluster/edit-cluster/template.html

```diff
@@ -96,7 +96,8 @@
                        formControlName="podNodeSelectorAdmissionPluginConfig"
                        fxFlex="100"></km-label-form>
         <div *ngIf="isPluginEnabled(admissionPlugin.EventRateLimit)">
-          <km-wizard-cluster-event-rate-limit [formControl]="form.get(Controls.EventRateLimitConfig)" [eventRateLimitConfig]="eventRateLimitConfig"></km-wizard-cluster-event-rate-limit>
+          <km-wizard-cluster-event-rate-limit [formControl]="form.get(Controls.EventRateLimitConfig)"
```

Comment:
This has already been fixed in https://github.com/kubermatic/dashboard/pull/4800 so you can revert this.

Reply:
This is happening when I run `npm run fix`, if I don't run it, pipeline will fail.

Reply:
Pipeline won't fail because of this particular case.